datalad/distribution/publish.py

```diff
@@ -292,14 +353,19 @@ def __call__(
 
         if dataset and since:
```

Comment:
To be more clear: By ignoring `since` if it is `None`, it implicitly defaults to local(!) HEAD, instead of the remote state, I think.